app/components/review_policy_class_link_component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ReviewPolicyClassLinkComponent < ViewComponent::Base
+  def initialize(policy_class:)
+    @policy_class = policy_class
+  end
+
+  attr_reader :policy_class
+
+  def link_path
+    case policy_class&.review_policy_class&.status
+    when "complete"
+      planning_application_review_policy_class_path(policy_class.planning_application, policy_class)
+    else
+      edit_planning_application_review_policy_class_path(policy_class.planning_application, policy_class)
+    end
+  end
```

Comment:
YAY components 😍

Reply:
I wonder if there's some way to namespace this one?

`ReveiwPolicyClasses::LinkComponent`?

Reply:
Or potentially you could create `TaskListRow::ReviewPolicyClass` component, with renders the whole `<li>` element? I can see a case for a bunch of `TaskListRow` components, which would mean we could gradually unburden `PlanningApplicationPresenter` 🤔

Reply:
although this is all just really potential future refactoring!

Reply:
RE: I wonder if there's some way to namespace this one?
      ReviewPolicyClasses::LinkComponent?
      
I wondered if there would there would be a "link" component like there's a "tag"component but wasn't very confident on that. So I thought about getting a few and then deciding what to do.

If I had to bet it would end up namespaced under ReviewPolicyClasses... Thoughts... am I just lazy?😴 Is procrastinating a super power? 🤔